### ISSUE TITLE
Make activities for the stuff in the Title bar. (Or add more)

### ISSUE THREAD

**Issue by zeokav** [zeokav/InstantMusic#13]
That ^

**Comment by raamish** [comment]
it's giving some error , are you sure your code is correct in the common.hpp ?

**Comment by zeokav** [comment]
What error?

**Comment by raamish** [comment]
That zeokav was 360 quickscoped last night.

**Comment by zeokav** [comment]
Yeah so to fix that you buy a monkey, teach it some tango and then dance with it like there's no other tomorrow. Tell me if it got fixed. :)
